components/Map/MapBase.vue

```diff
@@ -220,7 +244,7 @@ export default defineNuxtComponent({
         tolerance: 0.6,
         data: {
           type: 'FeatureCollection',
-          features,
+          features: this.featuresPrepare(this.features),
```

Comment:
Why not using `features` from `initPoiLayer` args ? instead of `this.features` ?

Reply:
Yes, looks like it need to be fixed.